pages/trouw-opties/index.tsx

```diff
@@ -61,11 +60,9 @@ export default function MultistepForm1() {
               <PageHeaderTemplate />
             </PageHeader>
             <PageContent>
-              <ButtonGroup>
-                <ButtonLink href="/" appearance="subtle-button">
-                  ← Terug
-                </ButtonLink>
-              </ButtonGroup>
+              <BackLink href="/" appearance="subtle-button">
```

Comment:
Is the backlink ever not a `subtle-button`? Because I think this prop might not have to be available for Backlink :)

Reply:
Nice spot, that wasn't supposed to be there, fixed! :D